packages/color-modes/src/index.tsx

```diff
@@ -131,8 +131,10 @@ export const ColorModeProvider: React.FC = ({ children }) => {
 
   if (theme.useCustomProperties !== false) {
     emotionTheme.colors = toCustomProperties(emotionTheme.colors, 'colors')
+    emotionTheme.original = {
```

Comment:
There's a tiny risk that users will have a key named "original" in their themes, what makes it a breaking change.
A use-case for a Symbol perhaps? We could keep this more "private".